skimage/filters/_gaussian.py

```diff
@@ -19,8 +19,8 @@ def _convert_input(image, preserve_range):
 
 
 def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
-             multichannel=None, preserve_range=False):
-    """Multi-dimensional Gaussian filter
+             multichannel=None, preserve_range=False, truncate=4.0):
```

Comment:
I understand our agreement on this, but I actually think that this should not require a deprecation procedure. Let's keep it as is, and discuss in the corresponding SKIEP :smile: .
P.S. How do others (numpy/scipy devs) manage such kind of cases?